### PR TITLE
Minor updates in MD files

### PR DESCRIPTION
**Pull request by alanyee**
-Made explicit HTTPS calls
-Minor edits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@
 | [![Build Status](https://ci.tensorflow.org/buildStatus/icon?job=tensorflow-master-cpu)](https://ci.tensorflow.org/job/tensorflow-master-cpu) | [![Build Status](https://ci.tensorflow.org/buildStatus/icon?job=tensorflow-master-linux-gpu)](https://ci.tensorflow.org/job/tensorflow-master-linux-gpu) | [![Build Status](https://ci.tensorflow.org/buildStatus/icon?job=tensorflow-master-mac)](https://ci.tensorflow.org/job/tensorflow-master-mac) | [![Build Status](https://ci.tensorflow.org/buildStatus/icon?job=tensorflow-master-win-cmake-py)](https://ci.tensorflow.org/job/tensorflow-master-win-cmake-py) | [![Build Status](https://ci.tensorflow.org/buildStatus/icon?job=tensorflow-master-android)](https://ci.tensorflow.org/job/tensorflow-master-android) |
 
 **TensorFlow** is an open source software library for numerical computation using
-data flow graphs.  Nodes in the graph represent mathematical operations, while
+data flow graphs.  The graph nodes represent mathematical operations, while
```
Comment (vrv) [inline README.md]: This doesn't seem any better than what was there before.

File: README.md [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
 the graph edges represent the multidimensional data arrays (tensors) that flow
-between them.  This flexible architecture lets you deploy computation to one
-or more CPUs or GPUs in a desktop, server, or mobile device without rewriting
+between them.  This flexible architecture lets you deploy computation to at least
+one CPUs or GPUs in a desktop, server, or mobile device without rewriting
```
Comment (vrv) [inline README.md]: I think I prefer 'one or more'.

**Review by vrv — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (alanyee) [inline README.md]: I prefer the parallelism as "the graph edges" is used later on in the sentence.

File: README.md (hunk 2 above)
Comment (alanyee) [inline README.md]: Ah, I just dislike that many or's being used in a sentence within such a close distance to each other.

File: README.md (hunk 1 above)
Comment (vrv) [inline README.md]: Good point, this is okay.

**Review by vrv — APPROVED** [review]

**Pull request merged**